spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1055,12 +1265,11 @@ protected Table createTable(int files) {
     return table;
   }
 
-  protected Table createTablePartitioned(int partitions, int files) {
+  protected Table createTablePartitioned(int partitions, int files, Map<String, String> options) {
```

Comment:
Instead of adding a parameter and changing all the usages, it may be cleaner to add another method
```java
protected Table createTablePartitioned(int partitions, int files, Map<String, String> options) {
all this code
}
```

And keep a version 
```java
 protected Table createTablePartitioned(int partitions, int files) {
     createTablePartitioned(partitions, files, Maps.newHashMap())
 }
```

Reply:
Sounds great!